components/clarinet-format/src/formatter/mod.rs

```diff
@@ -941,7 +1020,7 @@ impl<'a> Aggregator<'a> {
                 _ => value.to_string(),
             },
             PreSymbolicExpressionType::List(ref items) => {
-                self.format_list(items, previous_indentation)
+                self.display_list(items, previous_indentation)
```

Comment:
this is nasty duplication but if we fall into the `display_pse` branch we want to format it like before. If the atom is a `ListCons` we use the new `format_list`